### PR TITLE
Work on the copy a bit.

### PR DESCRIPTION
**Pull request by mads-hartmann**
This is mostly to make sure we're moving in the right directly. So the
README describes a bunch of features we don't have yet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,103 @@
-# hatch
+# hatch.sh
```
Comment (mads-hartmann) [inline README.md]: Can't figure out if we should call it hatch.sh everywhere or not :P

Comment (skovhus) [inline README.md]: I guess .sh makes it easier to google. But the executable would still be hatch.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,103 @@
...
+    brew install hatch
+
+Otherwise you can use pip
+
+    pip install hatch-cli
+
+## Background
+
+Amazon Web Services provides an overwhelming number of services and tools that
+well help you build your products. These services are highly configurable and
```
Comment (skovhus) [inline README.md]: Will

File: README.md [hunk 3]
```diff
@@ -1,36 +1,103 @@
...
 ### Static Websites
 
-This uses S3, CloudFront, and Route53.
+Hatch doesn't care if you're hand-writing your HTML or if you're using a build
+tool like webpack. We just need the static files and we'll put them online. This
+is achieved using S3, CloudFront and Route53.
+
+Create your website
+
+    hatch website create
```
Comment (skovhus) [inline README.md]: Init might be a better name for this. What is the convention?

**Pull request merged**